cloudant-client/src/test/java/com/cloudant/tests/SessionInterceptorExpiryTests.java

```diff
@@ -139,7 +139,7 @@ public void setupSessionInterceptor(boolean okUsable, String sessionPath) {
             rpInterceptor = ci;
         } else if (sessionPath.equals("/_iam_session")) {
             // Set the endpoint value before each test
-            IamCookieInterceptor ici = new IamCookieInterceptor("apikey", baseUrl);
+            IamCookieInterceptor ici = new IamCookieInterceptor("apikey", baseUrl, null);
```

Comment:
If you retain the original ctor as well then this change is not needed.

Reply:
Fixed in a1c95fad4ccf0e845fe4649613fc184c33168220